.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: ruby
 rvm:
   - 2.1.0
+
+branches:
```

Comment:
imho this should be configured in the travis configuration settings, not in the travis file, otherwise we constantly need to update that with branches that get added and retired. and then we would need to update it in all branches I guess?